addons/website_slides/static/src/js/slides_slide_archive.js

```diff
@@ -30,6 +30,14 @@ var SlideArchiveDialog = Dialog.extend({
         this.slideId = this.$slideTarget.data('slideId');
         this._super(parent, options);
     },
+    _displayCategoryEmptyFlag: function (){
+        var categorySlideList = $(this.$slideTarget.parents('li'));
+        var categoryId = categorySlideList.data('categoryId');
+        var slideList = $('.o_wslides_slide_list[data-category-id='+ categoryId +']').find('.o_wslides_js_list_item');
+        if (slideList.length === 0){
+            $('.o_wslides_slide_list_category_header[data-category-id="'+ categoryId +'"] > div:first-child').append('<small class="ml-1 text-muted">Empty</small>');
+        }
```

Comment:
Can't you trigger_up an event to the list that will call `_checkForEmptySections` ?

That feels like a lot of duplicated code.

Reply:
I can't seem to get the trigger_up to work in this case. I can trigger it up to the public widget who creates the dialog but apparently can't make it bubble up to the list widget itself.

This will be completely removed in the task about preventing refresh on slide upload: It will then trigger_up an event to a shiny new content management widget which will be responsible for manipulating the dom and attaching all the widgets to their dom elements.

Reply:
Well that's annoying.

I'm debating with myself how we should have done this.
Maybe a master widget on top of everything that handles the communication between the sub-widgets like list and delete?
Nothing we can do for stable anyway...

Then please:
- apply the same mechanism with the block `'<small class="ml-1 text-muted">Empty</small>'` as explained in my other comment
- prefix your variables containing jQuery elements with $
- use `closest('li')` instead of `$(this.$slideTarget.parents('li'))` if possible